### PR TITLE
completed assignment 1 and 1-3 labs

### PR DESCRIPTION
**Pull request by lthurair**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
added code to labs and assignment 1
## What did you learn from the changes you have made?
learned how to create neural network
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
n/a
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
yes, creating neural network layers
## How were these changes tested?
results displayed and plotted
## A reference to a related issue in your repository (if applicable)
n/a
## Checklist
- [X] I can confirm that my changes are working as intended

### CODE REVIEW

File: 01_materials/labs/lab_1.ipynb
Comment (tianyi21) [inline 01_materials/labs/lab_1.ipynb:1]: - b: Please always reinitialize your model with `model = Sequential() ...` to reset all model parameters in addition to `model.compile()`. Or, you are continuing the training from the previous trained state with a different model setup. You may check this with the accuracy of the first epoch.

**Review by tianyi21 — APPROVED** [review]
Hi Lisa!

Thank you for submitting your DL assignments. Please refer to my comments attached to each file.

Your A1 and Lab 1-3 are now complete.

Thanks,
Tianyi [LS]
